cpp/ps0005/problem5.cpp

```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+using namespace std;
+int main ()
+{
+    // prompting user for 2 operands and an operator 1.5/2
+    double operand1, operand2, result;
+    char operator_6;
+    cout << "Enter the first operand" << endl;
+    cin >> operand1;
+    cout << "Enter the second operand" << endl;
+    cin >> operand2;
+    cout << "Enter the operator(+,-,%,*,/) " << endl;
+    cin >> operator_6;
```

Comment:
- [x] declared variables to store the user input
- [ ] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input